### PR TITLE
Aws3

### PR DESCRIPTION
**Pull request by MohamadD**

### CODE REVIEW

**Comment by MohamadD** [comment]
This is the backend of  AW3

**Comment by MohamadD** [comment]
remove config.env from the branch

File: src/app.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 const express = require("express");
 const bodyParser = require("body-parser");
 const cors = require("cors");
+
 const apiRoutes = require("./routes/api");
 const adminRoutes = require("./routes/admin");
 
 const app = express();
+app.engine("html", require("ejs").renderFile);
```
Comment (Karyum) [inline src/app.js]: this needs to be removed we dont use an engine

File: src/routes/api.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
 const express = require("express");
 const router = express.Router();
 const { businesses, businessesId, newReview } = require("../controllers/api");
+const { aw3 } = require("../controllers/aw3");
```
Comment (Karyum) [inline src/routes/api.js]: give it a name better than `aw3` it's not a very clear name

File: src/app.js (hunk 1 above)
Comment (MohamadD) [reply src/app.js]: fixed

File: src/routes/api.js (hunk 2 above)
Comment (MohamadD) [reply src/routes/api.js]: fixed

**Pull request merged**